app/src/main/res/values/strings.xml

```diff
@@ -151,4 +151,5 @@
     <string name="connecting_to_server">Connecting to server</string>
     <string name="refreshing">Refreshing</string>
     <string name="logging_out">Logging out</string>
+    <string name="fetch_edit_message">Preparing message for Edit</string>
```

Comment:
we might consider using "Preparing message for Edit" which better matches the zulip capitalization scheme?